apps/examples.nameguard.io/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# nameguard-examples
+
+## null
```

Comment:
Should something go here?

Reply:
No, don't touch these. Managed by changesets.

Reply:
Removed now, thanks to `ignore`-d packages.